static/src/stylesheets/layout/new-header/_new-header.scss

```diff
@@ -0,0 +1,65 @@
+/* When the menu is open this class is added to the html to prevent users
+from scrolling */
+.nav-is-open {
+    overflow: hidden;
+    width: 100%;
+}
+
+.new-header {
+    background-color: $guardian-brand;
+    position: relative;
+
+    @include mq(tablet) {
+        display: none;
+    }
+}
+
+.new-header__inner {
+    align-items: flex-start;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+}
+
+.new-header__support {
+    left: ($gs-gutter / 4) - 1px;
+    position: absolute;
+
+    @include mq(mobileLandscape) {
+        left: $gs-gutter / 2;
+    }
+}
+
+.new-header__logo {
+    margin-left: auto;
+}
+
+.new-header__logo .inline-logo__svg {
```

Comment:
Yes, will do. I left it that way, because the inline functions breaks the BEM naming system (produces `.header__logo__svg`), but since it's all over the place I guess you are right.